pkg/pillar/cmd/zedagent/radiosilence.go

```diff
@@ -74,12 +74,25 @@ func radioPOSTTask(ctx *getconfigContext) {
 		}
 		config := getRadioConfig(ctx, status)
 		if config != nil {
-			if config.RadioSilence != ctx.radioSilence.Imposed {
-				ctx.radioSilence.Imposed = config.RadioSilence
-				ctx.radioSilence.ChangeInProgress = true
+			var configErr string
+			if config.RadioSilence && ctx.updateInprogress {
+				configErr = "Imposing radio silence during EVE update testing is not allowed"
+			}
```

Comment:
Will this check be re-done each time we fetch the radio config from the local profile server?

Reply:
Yes, so eventually the error disappears.